packages/lib-classifier/src/store/SubjectStore.js

```diff
@@ -1,13 +1,30 @@
 import asyncStates from '@zooniverse/async-states'
 import { autorun } from 'mobx'
-import { addDisposer, addMiddleware, flow, getRoot, isValidReference, onPatch, tryReference, types } from 'mobx-state-tree'
+import { addDisposer, addMiddleware, flow, getRoot, getType, isValidReference, onPatch, tryReference, types } from 'mobx-state-tree'
 import { getBearerToken } from './utils'
 import { filterByLabel, filters } from '../components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal'
 import ResourceStore from './ResourceStore'
 import Subject from './Subject'
+import SingleImageSubject from './SingleImageSubject'
+import SubjectGroup from './SubjectGroup'
```

Comment:
The subject models could be further reorganised so that these lines become
```js
import { Subject, SingleImageSubject, SubjectGroup } from './subjects'
```